examples/models/object_detection/yolox/pascal_voc/train.py

```diff
@@ -0,0 +1,456 @@
+import resource
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from tensorflow import keras
+
+import keras_cv
+
+low, high = resource.getrlimit(resource.RLIMIT_NOFILE)
+resource.setrlimit(resource.RLIMIT_NOFILE, (high, high))
+
+BATCH_SIZE = 8
+EPOCHS = 10
+CHECKPOINT_PATH = "checkpoint/"
+
+class_ids = [
+    "Aeroplane",
+    "Bicycle",
+    "Bird",
+    "Boat",
+    "Bottle",
+    "Bus",
+    "Car",
+    "Cat",
+    "Chair",
+    "Cow",
+    "Dining Table",
+    "Dog",
+    "Horse",
+    "Motorbike",
+    "Person",
+    "Potted Plant",
+    "Sheep",
+    "Sofa",
+    "Train",
+    "Tvmonitor",
+    "Total",
+]
+class_mapping = dict(zip(range(len(class_ids)), class_ids))
```

Comment:
if this is not used, then remove it? We can find a way to populate this info from dataset